.github/workflows/main.yml

```diff
@@ -62,6 +62,8 @@ jobs:
         rustup default stable
         rustup component add rustfmt
     - run: cargo fmt -- --check
```

Comment:
Maybe a note saying that `--all` doesn't work because we're not using Cargo workspaces?

Reply:
will do. Perhaps in the next iteration we should add those?  I didn't want to make all changes at once :)